src/utils/session-helper.test.js

```diff
@@ -1,6 +1,6 @@
+import { expect, vi, it, describe } from 'vitest';
```

Comment:
Removed assert for now, since it's not being used.

Reply:
I don't know testing, but even after an `npm install` I'm still left with this error here:
<img width="601" alt="Screenshot 2023-05-13 at 7 50 52 AM" src="https://github.com/codeforpdx/PASS/assets/71395931/b5207ef3-81d3-4062-8c3c-8d47a29a7aeb">
Doesn't seem to know/find `vitest` even though it exists in the `node_modules`

Reply:
I'm not sure about problem. I'm not seeing the same lint error from that file.

Reply:
Maybe try running a clean install?
`npm ci`